### PR TITLE
Exec copy for the week

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/votes/Executive vote - January 28, 2022.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+* **1,069,250 DAI** will be transferred to the [TechOps Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP29) at 0x2dC0420A736D1F40893B9481D8968E4D7424bC0B as a result of their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP53).
+* **6,966,070 DAI** will be transferred to the [Oracles Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13) at 0x2B6180b413511ce6e3DA967Ec503b2Cc19B78Db6 as a result of their [oracle gas budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP45).
+* **1,805,407 DAI** will be transferred to the [Oracles Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP13) at 0x1A5B692029b157df517b7d21a32c8490b8692b0f as a result of their [oracle gas budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP45). This is intended to be used as emergency funding for MakerDAO's oracles.
+
+### Core Unit DAI Budget Streams
+As per multiple successful budget proposals (linked below), total of **11,096,946 DAI** will be streamed to a number of Core Units over the next 6-12 months if this executive proposal passes.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - January 28, 2022.md:44]: "a total of"?

File: governance/votes/Executive vote - January 28, 2022.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Core Unit DAI Budget Transfers
+
+As per multiple successful budget proposals (linked below), total of **10,070,519 DAI** will be transferred to a number of Core Units if this executive proposal passes.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - January 28, 2022.md:37]: "a total of"?

File: governance/votes/Executive vote - January 28, 2022.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+* **240 MKR** will be streamed to SebVentures at 0xcc81578d163a04ea8d2eae6904d0c8e61a84e1bb starting 2021-04-01 and ending at 2024-04-01 with a cliff date of 2022-04-01.
+
+### Revoking Content Production DAI Budget
+As a result of this successful [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP49), the [Content Production Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP5) budget will be revoked if this executive proposal passes. Note that when a token stream is revoked, it will prevent new tokens accruing to the recipient, but will still allow the recipient access to tokens that have already accrued.
+
+### Replacing the Real World Finance DAI Budget Stream
+Given that the facilitator for the [Real World Finance Core Unit](https://mips.makerdao.com/mips/details/MIP39c2SP1) has changed, we recieved a request from the new facilitator to adjust the implementation of the existing Real World Finance DAI budget stream. This request is detailed here. Given that this is an administrative change and budget amounts remain the same, we elected to include it in this executive proposal. If this executive proposals passes the following will occur.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - January 28, 2022.md:60]: "This request is detailed here."

I'm assuming this sentence is meant to contain a hyperlink?

Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - January 28, 2022.md:60]: Also, link to will's onboarding MIP maybe? Might be unnecessary

File: governance/votes/Executive vote - January 28, 2022.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+* The existing Real World Finance DAI budget stream will be revoked.
+* A replacement stream will be setup, streaming **1,705,000 DAI** to 0x96d7b01Cc25B141520C717fa369844d34FF116ec starting at 2022-02-01 and ending at 2022-12-31.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive vote - January 28, 2022.md:62]: Maybe link to the budget MIP so we can see where the number came from?

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Just a few suggestions. 

All the addresses in the text match the spreadsheet.

Just one question, when we do the delegate comp we've been linking to the etherscan page for addresses. Should we do that for these proposals as well? Again, might be unnecessary work.

File: governance/votes/Executive vote - January 28, 2022.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - January 28, 2022.md:60]: Yes, not sure we have one yet.

File: governance/votes/Executive vote - January 28, 2022.md (hunk 4 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - January 28, 2022.md:62]: Think I'll leave this out. Important part is that it comes to the same amount, and Will's post should be illumination enough.

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM!

**Pull request merged**